### ISSUE TITLE
Laravel 7 installation failed

### ISSUE THREAD

**Issue by ed-fruty** [Chatter-Laravel/core#22]
I'm using fresh laravel 7 version and added larave/ui package.

```json
"laravel/framework": "^7.0",
"laravel/ui": "^2.0",
```

From the docs I read that I need to use `dev-master` branch for Laravel 7.
When I try to add chatter, I got an error:

```
composer require "chatter-laravel/core:dev-master"
./composer.json has been updated                                     
Loading composer repositories with package information                                                                                                   Updating dependencies (including require-dev)         Package operations: 20 installs, 0 updates, 0 removals
  - Installing paragonie/random_compat (v9.99.99): Loading from cache
  - Installing cocur/slugify (v4.0.0): Loading from cache
  - Installing cviebrock/eloquent-sluggable (7.0.1): Loading from cache
  - Installing lcobucci/jwt (3.3.2): Loading from cache
  - Installing psr/http-factory (1.0.1): Loading from cache
  - Installing laminas/laminas-zendframework-bridge (1.0.4): Loading from cache
  - Installing laminas/laminas-diactoros (2.3.0): Loading from cache
  - Installing laravel-frontend-presets/tailwindcss (4.1.0): Loading from cache
  - Installing symfony/psr-http-message-bridge (v2.0.0): Loading from cache
  - Installing phpseclib/phpseclib (2.0.27): Loading from cache
  - Installing php-http/message-factory (v1.0.2): Loading from cache
  - Installing nyholm/psr7 (1.3.0): Loading from cache
  - Installing defuse/php-encryption (v2.2.1): Loading from cache
  - Installing league/event (2.2.0): Loading from cache
  - Installing league/oauth2-server (8.1.0): Loading from cache
  - Installing firebase/php-jwt (v5.2.0): Loading from cache
  - Installing laravel/passport (v8.5.0): Loading from cache
  - Installing ezyang/htmlpurifier (v4.12.0): Loading from cache
  - Installing mews/purifier (3.2.2): Loading from cache
  - Installing chatter-laravel/core (dev-master 3c55d0b): Cloning 3c55d0b449 from cache
paragonie/random_compat suggests installing ext-libsodium (Provides a modern crypto API that can be used to generate random bytes.)
phpseclib/phpseclib suggests installing ext-libsodium (SSH2/SFTP can make use of some algorithms provided by the libsodium-php extension.)
mews/purifier suggests installing laravel/lumen-framework (To test the Lumen bindings)
Writing lock file
Generating optimized autoload files
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan package:discover --ansi

In ChatterInstallCommand.php line 13:
                                                                 
  Class 'Illuminate\Foundation\Console\PresetCommand' not found  
                                                                 

Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 1

Installation failed, reverting ./composer.json to its original content.

```

This command (PresetCommand) has been removed in Laravel 7 (In laravel 6 it's present).
Do you have a patch for integration with 7-th version ?

**Comment by jcrann** [comment]
Same issue.

**Comment by sbarbat** [comment]
No patch to run it on Laravel 7 yet, maybe that's something you can help with? 👍 😄

**Comment by sbarbat** [comment]
This is been fixed by [PR#26](https://github.com/Chatter-Laravel/core/pull/26). Can you try it again updating your
